cmd/generate_test.go

```diff
@@ -31,18 +31,11 @@ const (
 				{
 					"availabilityZones": null,
 					"count": 1,
-					"enableAutoScaling": null,
-					"maxCount": null,
-					"minCount": null,
 					"name": "nodepool1",
-					"orchestratorVersion": "1.19.3",
```

Comment:
Just to clarify, are we removing agentPoolProfile.orchestratorVersion because we have the same field as properties. orchestratorProfile.orchestratorVersion already present?

Reply:
The orchestratorVersion version in AgentPoolProfile is never used in AgentBaker, thus I removed it.